en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -1,11 +1,22 @@
 = MuleSoft Composer Release Notes
+:page-revdate: February 8, 2023
 
 Use MuleSoft Composer to build connected experiences faster. You don't need to wait for development resources to unlock data, integrate apps, and automate business processes that create real customer value.
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues for MuleSoft Composer] for a list of known issues by date.
 
+== February 8, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]
+
+[%header%autowidth.spread]
+|===
+|Issue Resolution |ID
+|Previously, when opening a flow, all steps of the flow were open by default. Now, when you open a flow, all steps aside from If/Else Blocks and For Each Loops are collapsed to provide a navigable outline of the flow |W-12515426
```

Comment:
Was this really a bug? It seems like an enhancement. 

IMO, you can leave the first line out entirely. Your call though. 

Now, when you open a flow, all steps except **If/Else Blocks** and **For Each Loops**, are collapsed to provide a navigable outline of the flow.